meshapp/src/pages/Profiles/ProfileAccordion.tsx

```diff
@@ -68,6 +68,7 @@ export default function ProfileAccordion(props: {
   alwaysOpen?: boolean;
   editHandler?: Function;
   deleteHandler?: Function;
+  viewOnly?: boolean | undefined;
```

Comment:
Is there a reason this an some others are able to be `undefined`?

Reply:
I did this since I thought it would be redundant and tedious to have to insert a value for viewOnly for other accordions where their purpose is to be edited; I can change this if you want me to.

Reply:
Oh okay that makes sense, nah I think it's fine, just was curious as to why this one had the option... but I think that as long as a bunch of other properties don't do the same thing it shouldn't matter...